### PR TITLE
feat(spin): Build/Publish Spin CLI for Windows

### PR DESCRIPTION
**Pull request by brantburnett**

### CODE REVIEW

**Comment by brantburnett** [comment]
I have successfully tested the build_spin command to ensure it works correctly. However, I'm unsure how to test the publish_spin command.

File: dev/buildtool/spin_commands.py [hunk 1]
```diff
@@ -46,10 +46,16 @@ class DistArch(namedtuple('DistArch', ['dist', 'arch'])):
   def __repl__(self):
     return '{},{}'.format(self.dist, self.arch)
 
+  def get_filename(self):
```
Comment (jtk54) [inline dev/buildtool/spin_commands.py]: You could shorten this by renaming the function to `filename` and annotating the function with `@property` e.g. https://github.com/spinnaker/spinnaker/blob/ae4006af07c08fc9259dd7c071bc7a39571cc6f9/dev/buildtool/base_metrics.py#L28.

Comment (brantburnett) [reply dev/buildtool/spin_commands.py]: Updated

**Comment by jtk54** [comment]
Thanks for the contribution.

**Pull request merged**